tests/cross_functional/system_test/test_clone_deletion_after_max_cluster_space_utilization.py

```diff
@@ -132,42 +130,55 @@ def test_clone_deletion_after_max_cluster_space_utilization(
         )
         clones_list = []
 
-        for clone_num in range(self.num_of_clones + 2):
-            logger.info(f"Start creation of clone number {clone_num}.")
-            cloned_pvc_obj = pvc_clone_factory(
-                self.pvc_obj, storageclass=self.pvc_obj.backed_sc, timeout=600
+        # Function to create clones
+        def create_clones(num_of_clones, start_num=0):
```

Comment:
cam we define this function outside of the test function?

Reply:
Yes we can define . If I don't find any necessity so kept as local.